src/main/java/com/lpvs/util/LPVSPayloadUtil.java

```diff
@@ -72,12 +73,12 @@ public static LPVSLicense convertOsoriDbResponseToLicense(String payload) {
             lic.setAccess("UNREVIEWED");
 
             List<String> nicknameList = new ArrayList<>();
-            detailInfoArray
-                    .get(0)
-                    .getAsJsonObject()
-                    .get("nicknameList")
-                    .getAsJsonArray()
-                    .forEach(element -> nicknameList.add(element.getAsString()));
+            JsonElement nicknameListArray =
+                    detailInfoArray.get(0).getAsJsonObject().get("nicknameList");
+            if (nicknameListArray != null && nicknameListArray.isJsonArray()) {
+                JsonArray nicknameArray = nicknameListArray.getAsJsonArray();
+                nicknameArray.forEach(element -> nicknameList.add(element.getAsString()));
```

Comment:
I don't think it's necessary to make a local variable. What about the suggestion below?
```suggestion
                nicknameListArray.getAsJsonArray().forEach(element -> nicknameList.add(element.getAsString()));
```

Reply:
Agree.